std_msgs/Header header
int64 ctl_status                # ctl_status reading. 0 is interpreted as false


